roles/satellite.tf

```
@@ -393,9 +393,9 @@ output "eks_satellite_management_role_name" {
 }
 
 output "s3_replication_policy_name" {
-  value = aws_iam_policy.s3_replication[0].name
+  value = length(aws_iam_policy.s3_replication) > 0 ? aws_iam_policy.s3_replication[0].name : null
```

```suggestion
  value = local.is_satellite_regions_enabled ? aws_iam_policy.s3_replication[0].name : null
```

logic should match the iam_policy resource